docs/cloud-providers/azure/azure-provider.md

```diff
@@ -60,6 +60,8 @@ The steps to perform this action are listed below:
 1. Click `Grant admin consent for Default Directory`
 1. Click `Yes`
 
+Note: You might need `Global Administrator` role to `Grant admin consent for Default Directory`. Please contact the administrator of your Azure subscription. To check your role, go to `Azure Active Directory` -> `Roles and administrators`. You can find your role(s) by clicking on `Your Role (Preview)`
```

Comment:
in the UI i'm looking at, mine says "Your Role: Global administrator".  If you're already a Global administrator, does the UI not say "Your Role (Preview)", or am I looking in the wrong place?

![Screen Shot 2019-11-15 at 7 06 21 AM](https://user-images.githubusercontent.com/4313439/68953339-92e15e00-0776-11ea-87d6-f95fec41051f.png)

Reply:
You are right! But a user can have multiple roles. Preview shows only one role. If you have other roles, it looks like this (I think what you are saying should work for most of the cases but I have added the clicking part just in case):
![Screenshot from 2019-11-18 12-35-02](https://user-images.githubusercontent.com/34534103/69031407-1366bf80-0a00-11ea-9490-182b280d74b8.png)

Reply:
ah, got it! thank you for clarifying!